apps/test/netsim/NetSimLogger.js

```diff
@@ -15,8 +26,8 @@ var OldConsole = function () {
 };
 
 // Modern console with log, info, warn, error methods.
-var ModernConsole = function () {
-  var that = OldConsole();
```

Comment:
Heh, that wasn't sarcasm. I legitimately do love parasitic inheritance. I had no reason beyond that for calling it out